presto-main/src/main/java/com/facebook/presto/operator/OperatorStats.java

```diff
@@ -70,12 +77,15 @@ public OperatorStats(
             @JsonProperty("planNodeId") PlanNodeId planNodeId,
             @JsonProperty("operatorType") String operatorType,
 
+            @JsonProperty("totalDrivers") long totalDrivers,
+
             @JsonProperty("addInputCalls") long addInputCalls,
             @JsonProperty("addInputWall") Duration addInputWall,
             @JsonProperty("addInputCpu") Duration addInputCpu,
             @JsonProperty("addInputUser") Duration addInputUser,
             @JsonProperty("inputDataSize") DataSize inputDataSize,
             @JsonProperty("inputPositions") long inputPositions,
+            @JsonProperty("inputPositionsSquared") double inputPositionsSquared,
```

Comment:
This doesn't seem like the right place for this. I would just square `inputPositions` in whatever callers need the square